src/components/theme/getThemeType.ts

```diff
@@ -1,6 +1,6 @@
-import {LIGHT_THEMES} from './constants';
+import {DARK_THEMES} from './constants';
 import {RealTheme, ThemeType} from './types';
 
 export function getThemeType(theme: RealTheme): ThemeType {
```

Comment:
When storybook is launching `theme` argument is undefined. This leads to switching the light theme back and forth